org.civicrm.bootstrapcivicrm/CRM/Contribute/Form/ContributionCharts.php

```diff
@@ -198,6 +198,28 @@ public function postProcess() {
       }
       $values['size'] = array('xSize' => $xSize, 'ySize' => $ySize);
     }
+
+    // @custom Start custom code
+    $colors = array('#5A6779', '#E5807F', '#ECA67F', '#8EC68A', '#C096AA', '#9579A8', '#42B0CB');
+
+    $colorIndex = 0;
+    foreach ($chartData as $key => $value) {
+
+      $chartData[$key]['object']->bg_colour = '#FFFFFF';
+      $chartData[$key]['object']->y_axis->colour = '#edf2f3';
+      $chartData[$key]['object']->y_axis->{'grid-colour'} = '#FFFFFF';
+      $chartData[$key]['object']->x_axis->colour = '#edf2f3';
+      $chartData[$key]['object']->x_axis->{'grid-colour'} = '#edf2f3';
+
+      for ($i=0; $i < count($chartData[$key]['object']->elements); $i++) {
+        $chartData[$key]['object']->elements[$i]->colour = $colors[$colorIndex];
+        $chartData[$key]['object']->elements[$i]->type = 'bar';
+        $chartData[$key]['object']->elements[$i]->alpha = 1;
+        $colorIndex++;
```

Comment:
remove this line and do this instead : 

```php
$chartData[$key]['object']->elements[$i]->colour = $colors[++$colorIndex];
```

since you only using it once